aiohttp/client_reqrep.py

```diff
@@ -481,8 +481,6 @@ def update_proxy(
         proxy_auth: Optional[BasicAuth],
         proxy_headers: Optional[LooseHeaders],
     ) -> None:
-        if proxy and not proxy.scheme == "http":
-            raise ValueError("Only http proxies are supported")
```

Comment:
I think we need to add a warning here if asyncio doesn't yet support TLS-in-TLS with pointers to how to monkeypatch it.